deployer/templates/fluentd.yaml

```diff
@@ -343,4 +343,4 @@ parameters:
 -
   description: 'Fluentd buffer size limit'
```

Comment:
> I'm also wondering deployer/templates/fluentd.yaml is in-use or not...

No, it is not.  Ignore everything under `deployer/` except `scripts`, and those are only used for testing.